go.mod

```diff
@@ -1,10 +1,22 @@
-module github.com/gettaxi/newrelicutil
+module github.com/gettaxi/newrelicutil/v2
```

Comment:
Maybe let's also change the import path to `gtforge`?

Reply:
Good catch, missed that.

Reply:
Fixed https://github.com/gtforge/newrelicutil/pull/1/commits/fc74a0537175d0cd02ffdd4f7fa773a48a3871f0